tests/e2e/performance/test_vdbench_workload.py

```diff
@@ -48,6 +48,9 @@ def label_nodes(request, with_ocs):
 
     def teardown():
 
+        if not ceph_health_check():
```

Comment:
no need if and raise.
Just call ceph_health_check().  it will raise in case a failure

Reply:
Fixed it.